tools/ds-identify

```diff
@@ -630,14 +629,14 @@ read_datasource_list() {
             dslist=${_RET}
             ;;
     esac
-    if [ -z "$dslist" ] && check_config datasource_list; then
+    if [ -z "$dslist" ] && check_config "$key" && get_value "$key" "$_RET"; then
```

Comment:
If I understand this correctly, it checks that _something_ exists after the `datasource_list:`, correct? Even if it's just a space? Wouldn't `datasource_list: ` pass the check here? Can we instead check that there's at least an opening bracket somewhere on the line or some kind of non-whitespace?

Reply:
> If I understand this correctly, it checks that something exists after the datasource_list:, correct? Even if it's just a space? Wouldn't datasource_list: pass the check here?

Correct

> Can we instead check that there's at least an opening bracket somewhere on the line

This would break the `disable_vmware_customization` callsite, which expects a boolean not a list.

> or some kind of non-whitespace?

Yeah we can make this more strict.